integration/travis/wait-for-cook.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Usage: wait-for-cook.sh [PORT]
+#
+# Examples:
+#   timeout 60s wait-for-cook.sh 12321
+#   timeout 60s wait-for-cook.sh
+#
+# Waits for cook to be listening on the provided port (defaults to 12321).
```

Comment:
Why create a script instead of a function?  Also, `curl` supports retries.  See `--retry` and `--retry-delay` and `--retry-max-time`.  It seems like we could avoid this entire script and move the curl calls back into `run_integration.sh`.